numpy/conftest.py

```diff
@@ -89,6 +89,6 @@ def check_fpu_mode(request):
                                                                 new_mode))
 
 
-@pytest.fixture(autouse=True)
-def add_np(doctest_namespace):
-    doctest_namespace['np'] = numpy
+#@pytest.fixture(autouse=True)
+#def add_np(doctest_namespace):
+#    doctest_namespace['np'] = numpy
```

Comment:
```suggestion
@pytest.fixture(autouse=True)
def add_np(doctest_namespace):
    doctest_namespace['np'] = numpy
```
Woops, that change was accidentally commit (its necessary for pytest-leaks unfortunately).